util.py

```diff
@@ -0,0 +1,36 @@
+# MIT License
+#
+# Copyright (c) 2017 Huddle
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+import subprocess
+import sys
+
+
+def run_subprocess(args, error_msg, verbose):
+    completed_process = subprocess.run(args, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, shell=False)
+
+    if completed_process.returncode != 0:
+        print(completed_process.stdout.decode('utf-8'))
+        print(error_msg, file=sys.stderr)
+        exit()
```

Comment:
Do we always want to exit if a process fails? May want to continue and just report on the errors ... just a thought.

Reply:
After a little bit of discussion with Ian, we think the behvaiour that makes the most sense is that deploying multiple projects shouldn't cause each other to fail. However, within a project the order of packages is usually important, so if a `*Deploy.ps1` fails within a project then it shouldn't deploy any more packages/run any more scripts? 

If you agree, I might raise another issue for this.

Reply:
Yeah if one step fails fail that whole deployment but go on to process the rest of the deployments. 
Think it gets super annoying when everything fails cos some rookie in Babble breaks their section :p agree do that on a separate PR